dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/EventExporterTest.java

```diff
@@ -101,6 +103,9 @@ class EventExporterTest extends TrackerTest {
 
   private TrackedEntity trackedEntity;
 
+  private String occurredAtTimeStamp = "2019-01-25T12:10:38.100";
```

Comment:
I would inline them in the assertion as these values are only used in one test. Also in here its hard to know to what entity they belong to as there is no context.

Reply:
resolved